query-service-impl/src/main/java/org/hypertrace/core/query/service/projection/ProjectionTransformation.java

```diff
@@ -58,7 +58,7 @@
 
   private Single<List<Expression>> transformExpressionList(List<Expression> expressionList) {
     return Observable.fromIterable(expressionList)
-        .flatMapSingle(this::transformExpression)
+        .concatMapSingle(this::transformExpression)
```

Comment:
Would it be sequential instead of parallel now? Any impact on the speed or parallellism?

Reply:
Yes this makes it sequential. Should be negligible impact on speed - the only things that are async in this pipeline (attributes) are fetched together and cached so don't benefit from parallelism.